tests/network/probes.go

```diff
@@ -97,8 +97,8 @@ var _ = SIGDescribe("[ref_id:1182]Probes", func() {
 				serverStarter(vmi, readinessProbe, 1500)
 			} else {
 				By(specifyingVMReadinessProbe)
-				vmi = tests.NewRandomFedoraVMIWithGuestAgent()
-				vmi.Spec.ReadinessProbe = readinessProbe
+				vmi = libvmi.NewFedora(
+					withMasqueradeNetworkingAndFurtherUserConfig(withReadinessProbe(readinessProbe))...)
```

Comment:
I think you could have left it flat at some level, this seems too complicated to read (for me).

i.e:
```

vmi = libvmi.NewFedora(
    withReadinessProces,
    withMasqueradeNetworking()...,
)
```

Is that possible?